pkg/entities/entities_api.go

```diff
@@ -1890,14 +2308,16 @@ func (a *Entities) GetEntitySearchWithContext(
 	query string,
 	queryBuilder EntitySearchQueryBuilder,
 	sortBy []EntitySearchSortCriteria,
+	sortByWithDirection []SortCriterionWithDirection,
 ) (*EntitySearch, error) {
 
 	resp := entitySearchResponse{}
 	vars := map[string]interface{}{
-		"options":      options,
-		"query":        query,
-		"queryBuilder": queryBuilder,
-		"sortBy":       sortBy,
+		"options":             options,
+		"query":               query,
+		"queryBuilder":        queryBuilder,
+		"sortBy":              sortBy,
+		"sortByWithDirection": sortByWithDirection,
```

Comment:
@sanderblue few thoughts - unfortunately, we weren't able to have `sortByWithDirection` so far, so by having this change to support the addition of `sortByWithDirection` move forward (which of course, is Tutone generated)
- we will be breaking `terraform-provider-newrelic` / `newrelic-cli` stuff using this function (though the fix would just be to add one tiny argument, so maybe it's not technically "breaking" :) )
- the same, for direct consumers of the New Relic Go Client - though again, the fix is just to add an extra empty argument

Do you think the above need to be considered? Maybe not entirely, in my opinion, since the only customer facing impact caused would be for direct consumers of this function from the Go Client, but I'd like to know your thoughts - if we do have to consider, we might just need to undo this change, and ignore when tutone shows a drift adding this argument everytime we run a `tutone generate` - but all of this is just what I think, your call; let me know what you think.

Reply:
While this requires a change in the provider as well as in the CLI, this technically isn't a breaking change for those projects. I have a [PR](https://github.com/newrelic/terraform-provider-newrelic/pull/2697) to update the provider to support this change 🙂

Reply:
Yeah, agreed - which is why I mentioned in my comment too that this isn't supposed to be a "breaking" change :) my only concern is for those who could be using this function from the Go Client directly in other Go-based stuff, but I think this should be okay 👍 thanks for taking a look